sway-fmt/src/constants.rs

```diff
@@ -3,3 +3,7 @@
 // if it comes across a known pattern
 pub const NEW_LINE_PATTERN: &str = "+/+";
 pub const ALREADY_FORMATTED_LINE_PATTERN: &str = "---";
+
+// TODO there is a circular dep here between sway-server's definition of tab size and ours
+// we should just standardize when swayfmt.toml gets implemented
```

Comment:
There are good arguments for why an opinionated formatter rather than a configurable one is actually better, and I don't disagree with them necessarily: https://github.com/FuelLabs/sway/issues/605#issuecomment-1026310760

Might be worth standardizing this between the LSP server and formatter, with the former simply using the latter (or possibly removing this specific feature altogether from the LSP server). Can be saved for a future PR so long as it's tracked in an issue.

cc @JoshuaBatty

Reply:
that constant should not exist anymore since there are `FormattingOptions` that I've added recently
https://github.com/FuelLabs/sway/blob/master/sway-fmt/src/fmt.rs#L80